runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -809,6 +973,11 @@ J9::CodeGenerator::lowerTreeIfNeeded(
    TR_J9VMBase *fej9 = (TR_J9VMBase *)(self()->comp()->fe());
    OMR::CodeGeneratorConnector::lowerTreeIfNeeded(node, childNumberOfNode, parent, tt);
 
+   if (node->getOpCode().isCall())
+      {
+      lowerNonhelperCallIfNeeded(node, tt);
```

Comment:
According to Travis CI this seems to be missing a `self()->`

Reply:
Yes! It most certainly does.